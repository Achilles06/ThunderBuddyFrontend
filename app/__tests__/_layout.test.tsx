import { jest, describe, it, expect } from '@jest/globals';
import React from 'react';
import { render } from '@testing-library/react-native';
import RootLayout from '../_layout';

// Mock expo-router's Stack component
jest.mock('expo-router', () => ({
  Stack: () => null,
}));

describe('RootLayout Component', () => {
  it('renders without crashing', () => {
    const { toJSON } = render(<RootLayout />);
    expect(toJSON()).toMatchSnapshot();
  });
});