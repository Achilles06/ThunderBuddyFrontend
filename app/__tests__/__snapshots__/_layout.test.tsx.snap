// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RootLayout Component renders without crashing 1`] = `null`;
